assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -117,16 +124,64 @@ export default Ember.Controller.extend({
       .finally(() => this.set("loading", false));
   },
 
+  async _importQuery(file) {
+    const json = await this._readFileAsTextAsync(file);
+    const query = await this._parseQuery(json);
+    const record = this.store.createRecord("query", query);
+    const response = await record.save();
+    return response.target;
+  },
+
+  async _parseQuery(json) {
```

Comment:
I don't think this function needs to be `async`, nothing that is promise-related is happening here.

Reply:
Oh, this is redundant here, a good catch!